Makefile

```diff
@@ -9,6 +9,7 @@ SRC_DIR:=src
 TEST?=tests
 N_CPU?=4
 CONCRETE_PACKAGE_PATH=$(SRC_DIR)/concrete
+SPHINX_APIDOC_EXCLUDE=$(SRC_DIR)/concrete/ml/common/preprocessors.py
```

Comment:
what is this exactly ?

Reply:
for some reason sphinx was crashing for this file

Reply:
this can now be removed since we don't have sphinx anymore